tests/conftest.py

```diff
@@ -117,7 +117,11 @@ def exporter(tmpdir, crossbar):
         """
     Testport:
         NetworkSerialPort:
-          {host: 'localhost', port: 4000}
+          host: 'localhost'
+          port: 4000
+    Broken:
+        RawSerialPort:
+          port: 'none'
```

Comment:
Just to make sure I understand this: This lets `assert start_params['path'].startswith('/dev/')` raise an AssertionError. This is then caught and `broken` is set to 'start failed'. Correct?

Reply:
Yes, this was the easiest way to support a test for this code.